frontend/webpack.common.js

```diff
@@ -3,96 +3,96 @@ const MiniCssExtractPlugin = require('mini-css-extract-plugin');
 const HtmlWebpackPlugin = require('html-webpack-plugin');
 
 module.exports = {
-    target: 'web',
-    entry: {
-        app: './src/index.tsx'
-    },
-    output: {
-        // Add hash to make filename different depending on file content
-        // to enforce browsers to reload changes in production.
-        filename: '[name].[contenthash].js',
-        path: path.resolve(__dirname, 'dist'),
-    },
+  target: 'web',
+  entry: {
+    app: './src/index.tsx'
+  },
+  output: {
+    // Add hash to make filename different depending on file content
+    // to enforce browsers to reload changes in production.
+    filename: '[name].[contenthash].js',
+    path: path.resolve(__dirname, 'dist'),
+  },
 
-    resolve: {
-        alias: {
-            '~': path.resolve(__dirname, 'src/'),
-        },
-        // Add `.ts` and `.tsx` as a resolvable extension.
-        extensions: [".ts", ".tsx", ".js"]
+  resolve: {
+    alias: {
+      '~': path.resolve(__dirname, 'src/'),
     },
+    // Add `.ts` and `.tsx` as a resolvable extension.
+    extensions: [".ts", ".tsx", ".js"]
+  },
 
-    module: {
-        rules: [
-            // all files with a `.ts` or `.tsx` extension will be handled by `ts-loader`
-            {
-                test: /\.tsx?$/,
-                loader: "ts-loader"
-            },
-
-            {
-                test: /\.(sa|sc|c)ss$/,
-                use: [
-                    {
-                        loader: MiniCssExtractPlugin.loader,
-                        options: {},
-                    },
-                    'css-loader',
-                    'postcss-loader',
-                    'sass-loader',
-                ],
-            },
+  module: {
+    rules: [
+      // all files with a `.ts` or `.tsx` extension will be handled by `ts-loader`
+      {
+        test: /\.tsx?$/,
+        loader: "ts-loader"
+      },
 
-            {
-                test: /\.(png|jpg|jpeg|gif)$/i,
-                use: [
-                    {
-                        loader: 'url-loader',
-                        options: {
-                            limit: 10000,
-                        },
-                    }
-                ]
-            },
+      {
+        test: /\.(sa|sc|c)ss$/,
+        use: [
+          {
+            loader: MiniCssExtractPlugin.loader,
+            options: {},
+          },
+          'css-loader',
+          'postcss-loader',
```

Comment:
I have this warning. Is it a big deal:

```
warning " > postcss-loader@5.0.0" has unmet peer dependency "postcss@^7.0.0 || ^8.0.1".
```

?

Reply:
Seems OK:

```
$ webpack --config webpack.dev.js
(node:7410) [DEP_WEBPACK_COMPILATION_ASSETS] DeprecationWarning: Compilation.assets will be frozen in future, all modifications are deprecated.
BREAKING CHANGE: No more changes should happen to Compilation.assets after sealing the Compilation.
        Do changes to assets earlier, e. g. in Compilation.hooks.processAssets.
        Make sure to select an appropriate stage from Compilation.PROCESS_ASSETS_STAGE_*.
asset app.4165c3a16a5e6d6707bd.js 2.47 MiB [emitted] [immutable] (name: app)
asset index.html 343 bytes [emitted]
asset style.55dda2c204c4d7d5df0a.css 1 bytes [emitted] [immutable] (name: app)
Entrypoint app 2.47 MiB = style.55dda2c204c4d7d5df0a.css 1 bytes app.4165c3a16a5e6d6707bd.js 2.47 MiB
runtime modules 670 bytes 3 modules
cacheable modules 981 KiB
  modules by path ./node_modules/ 979 KiB
    modules by path ./node_modules/scheduler/ 31.8 KiB 4 modules
    modules by path ./node_modules/react/ 70.6 KiB 2 modules
    modules by path ./node_modules/react-dom/ 875 KiB 2 modules
    ./node_modules/object-assign/index.js 2.06 KiB [built] [code generated]
  modules by path ./src/ 1.7 KiB
    modules by path ./src/*.tsx 609 bytes
      ./src/index.tsx 195 bytes [built] [code generated]
      ./src/App.tsx 414 bytes [built] [code generated]
    ./src/index.scss 50 bytes [built] [code generated]
    ./src/assets/serokell.svg 1.06 KiB [built] [code generated]
css ./node_modules/css-loader/dist/cjs.js!./node_modules/postcss-loader/dist/cjs.js!./node_modules/sass-loader/dist/cjs.js!./src/index.scss 0 bytes [code generated]
webpack 5.21.2 compiled successfully in 3814 ms
Done in 4.73s.
```

Reply:
Not bit, it's causes by the fact latest PostCSS doesn't work with latest autoprefixer yet or something, so I had to downgrade PostCSS version